tests/typings/webdriverio/config.ts

```diff
@@ -79,7 +79,7 @@ const config: WebdriverIO.Config = {
             logName: 'test.log',
             videoName: 'test.mp4'
         }
-    }],
+    }] as WebDriver.DesiredCapabilities[],
```

Comment:
why do we need to do this now?

Reply:
because `capabilities` are defined as:

```ts
capabilities?: WebDriver.DesiredCapabilities[] | MultiRemoteCapabilities;
```

so this says that it is the first type and not multi remote caps.